jiant/evaluate.py

```diff
@@ -88,10 +88,8 @@ def evaluate(
                 batch = move_to_device(batch, cuda_device)
                 out = model.forward(task, batch, predict=True)
 
-            # We don't want diagnostic tasks to affect the micro and macro average.
```

Comment:
Why are we deleting this? Was this never actually used in the averages?

Reply:
We just don't use `n_examples` for diagnostic tasks later.